src/ucp/core/ucp_ep.c

```diff
@@ -461,7 +461,10 @@ ucs_status_t ucp_ep_create_server_accept(ucp_worker_h worker,
     /* coverity[overrun-local] */
     status = ucp_address_unpack(worker, sa_data + 1, addr_flags, &remote_addr);
     if (status != UCS_OK) {
-        goto out;
+        ucs_warn("failed to unpack address of client, status %s",
+                 ucs_status_string(status));
+        ucp_listener_reject(conn_request->listener, conn_request);
```

Comment:
dont you need to iface_reject if it's not cm flow?

Reply:
**ucp**_listener_reject cover both cases